README.md

```diff
@@ -91,10 +91,12 @@ Such questions arise frequently in customer segmentation (what is the effect of
 
 # News
 
-**March 6, 2020:** Release v0.7.0, see release notes [here](https://github.com/Microsoft/EconML/releases/tag/v0.7.0)
+**August 4, 2020:** Release v0.8.0b1, see release notes [here](https://github.com/Microsoft/EconML/releases/tag/v0.8.0b1)
```

Comment:
Why August 4th here?

Reply:
Whoops, supposed to be September.  I'll fix in an update.